scanner/src/test/assets/pigz-c-scanoss.json

```diff
@@ -0,0 +1,30 @@
+{
+  "pigz.c": [
+    {
+      "id": "file",
+      "lines": "all",
+      "oss_lines": "all",
+      "matched": "100%",
+      "vendor": "madler",
+      "component": "pigz",
+      "version": "v2.4",
+      "latest": "v2.4",
+      "url": "https://github.com/madler/pigz/archive/v2.4.tar.gz",
+      "file": "pigz-2.4/pigz.c",
+      "dependencies": [],
+      "licenses": [
+        {
+          "name": "Zlib",
+          "source": "file_header"
+        }
+      ],
+      "copyrights": [
+        {
+          "name": "Copyright (C) 2007-2017 Mark Adler",
+          "source": "file_header"
+        }
+      ],
+      "elapsed": "0.000907s"
+    }
+  ]
+}
```

Comment:
Please add a final newline to make Git(Hub) happy.

Reply:
done